### PR TITLE
Video header bidding support to RhythmOne bidder adapter

### PR DESCRIPTION
**Pull request by jstocker76**
## Type of change

- [ ] Bugfix
- [X] Feature
- [ ] New bidder adapter
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other

## Description of change
Added video header bidding support to RhythmOne bidder adapter

- test parameters for validating bids
```
{
	bidder: 'rhythmone',
	params: {
		endpoint: "//1r-qa-ads.rhythmxchange.com/rmp/{placementId}/0/{path}?z={zone}",
		placementId: "389633",
		zone: "csmith5",
		path: "mvo",
		debug: true,
		method:"get"
	}
}
```
- contact email of the adapter’s maintainer
- [ ] official adapter submission

### CODE REVIEW

File: src/adapters/rhythmone.js [hunk 1]
```diff
@@ -150,120 +159,77 @@ module.exports = function(bidManager, global, loader){
...
         query.push(encodeURIComponent(k)+"="+encodeURIComponent(v));
     }
+
+    p("domain", attempt(function(){
+        var d = global.document.location.ancestorOrigins;
+        if(d && d.length > 0)
+          return d[d.length-1];
+        return global.top.document.location.hostname;
+      },""));
+    p("title", attempt(function(){return global.top.document.title;},""));
```
Comment (jstocker76) [inline src/adapters/rhythmone.js]: The attempt function wraps this callback in a try/catch - somewhere around line 78

File: src/adapters/rhythmone.js [hunk 2]
```diff
@@ -150,120 +159,77 @@ module.exports = function(bidManager, global, loader){
...
+        v = v.join(",");
       if(typeof v !== "undefined")
         query.push(encodeURIComponent(k)+"="+encodeURIComponent(v));
     }
+
+    p("domain", attempt(function(){
+        var d = global.document.location.ancestorOrigins;
+        if(d && d.length > 0)
+          return d[d.length-1];
+        return global.top.document.location.hostname;
```
Comment (jstocker76) [inline src/adapters/rhythmone.js]: The attempt function wraps this callback in a try/catch - somewhere around line 78

File: src/adapters/rhythmone.js [hunk 3]
```diff
@@ -1,13 +1,12 @@
 var bidmanager = require('../bidmanager.js'),
   bidfactory = require('../bidfactory.js'),
-  utils = require('../utils.js'),
   CONSTANTS = require('../constants.json');
 
 import {ajax as ajax} from '../ajax';
 
-function track(debug, p1, p2, p3) {
+function track(debug) {
```
Comment (jstocker76) [inline src/adapters/rhythmone.js]: We use google analytics to test this bidder on some of our own sites, and use this function for that purpose.  Having to add it and all its references back, only to remove it before a push, repeatedly, would be cumbersome.

**Comment by jstocker76** [comment]
My bad, I've updated the bidder parameters to an endpoint and placement id combo that reliably return video bids

File: src/adapters/rhythmone.js (hunk 3 above)
Comment (matthewlane) [inline src/adapters/rhythmone.js]: Makes sense, thanks for clarifying

**Pull request merged**